scripts/src/main/resources/scripts/command/az

```diff
@@ -10,88 +10,45 @@ then
   exit
 fi
 
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
 
 function doSetup() {
-  if command -v az &> /dev/null
+  if [ "${1}" != "silent" ] || [[ ! $(command -v az) =~ ${DEVON_IDE_HOME}/software/ ]]
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
-    then
-      doEcho "Azure CLI is already installed at $(command -v az)"
-    fi
-  else
-    doRequireNotBatch
-    if doIsWindows
-    then
-      # Get leatest release
-      if [ -z "${AZ_VERSION}" ]
-      then
-        doEcho "Getting latest release..."
-        AZ_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
-      fi
-
-      doInstall "-" "${AZ_HOME}" "az" "${AZ_VERSION}"
-      if [ "${?}" = 0 ]
-      then
-        doPackageInstall "${AZ_HOME}"
-      fi
-    fi
+    doDevonCommand python
+    doRunCommand "pip3 install azure-cli"
+    setConfig
```

Comment:
I changed the command so that it works for the latest python command. In the latest python command you can't run `devon python pip`, but you can run pip individually.

Further in the python command still need to add the script folder to the path.
`PATH=/c/projects/devonfw-ide-scripts-2022.04.003/software/python/Scripts:$PATH`

Currently, if you run `devon az` when python is not installed, you have to restart the console again so that pip3 is recognized in the path.
`

Reply:
> In the latest python command you can't run `devon python pip`. 

Indeed. Thanks for pointing this out and sorry for that. Seems that I broke this. I created #839 and will fix this on Monday. Should be no big deal. You can then change this back to a single `doDevonCommand` again.
BTW: If you want to to a dependent setup always do with `setup silent`. Example:
https://github.com/devonfw/ide/blob/f1d26c141dc923bb3ab8f754cba4754e8fcc5278/scripts/src/main/resources/scripts/command/mvn#L23

> Further in the python command still need to add the script folder to the path.

I still disagree to that approach. This will only cause added complexity and future bugs.

However, I just tested and see what you mean. By default pip is still installing to the Scripts folder. This is hardcoded somewhere inside pip and totally undesired and violating obvious standard and on all platforms other than Windows it uses the `bin` folder instead.
Anyhow, this is good as it pointed me to a solution that seems to be a general improvement to me:
https://intellipaat.com/community/9428/install-a-python-package-into-a-different-directory-using-pip

So we can simply install `az` to `software/az` folder by providing the `--target=` option (we have to test, maybe the second option is better). Then we would again follow our own conventions and best-practices and have a folder for each tool inside `software`. This is much cleaner and as always "uninstalling" simply means delete the folder, done.

Reply:
I have added the azure installation with .pth file. Make sure that Python must be already installed or if not, that it comes to an error if the shell is not restarted after the Python installation. This may need to be adjusted in the Python command. The Python `Script` folder has only two scripts, which are created by reinstalling pywin32.

All other files belonging to Azure are stored in the `software/az` folder.

Reply:
After installation, the shell must also be restarted so that az is added in the path.

Reply:
Now working with just `doDevonCommand pip install azure-cli`.
However, this is causing that the software folder is called `azure-cli` instead of `az`.
I also observed other small issues.
I will create a new PR as replacement to be able to push the changes.